rpcd/playbooks/roles/rpc_maas/templates/galera_check.yaml.j2

```diff
@@ -34,3 +34,54 @@ alarms      :
             if (percentage(metric["mysql_current_connections"], metric["mysql_max_configured_connections"]) > {{ mysql_connection_warning_threshold }} ) {
                 return new AlarmStatus(WARNING, "Mysql connectons has exceeded {{ mysql_connection_warning_threshold }}% of configured maximum");
             }
+    open_file_size_limit_reached :
+        label                   : open_file_size_limit_reached--{{ ansible_hostname }}
+        notification_plan_id    : "{{ maas_notification_plan }}"
+        criteria                : |
+            :set consecutiveCount={{ maas_alarm_local_consecutive_count }}
+            if (metric["num_of_open_files"] > metric["open_files_limit"]) {
+                return new AlarmStatus(CRITICAL, "Mysql open files has exceeded open files limit");
+            }
+    innodb_row_lock_time_avg :
+        label                   : innodb_row_lock_time_avg--{{ ansible_hostname }}
+        notification_plan_id    : "{{ maas_notification_plan }}"
+        criteria                : |
+            :set consecutiveCount={{ maas_alarm_local_consecutive_count }}
+            if (metric["innodb_row_lock_time_avg"] > metric["innodb_row_lock_time_max"]) {
```

Comment:
You're right. I was thinking this criteria could catch any cases where it took an unusual amount of time getting the lock, but I do realize now that innodb_row_lock_time_max is maximum time in milliseconds spent getting an InnoDB row lock, which could be a value just *slightly* above the average at any given time.

I like your suggestion, but we still have to decided on a good threshold to set by default.